test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/MvcTagHelpersWebSite.MvcTagHelper_Home.OrderUsingHtmlHelpers.Encoded.html

```diff
@@ -0,0 +1,84 @@
+﻿
+
+
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <form action="HtmlEncode[[/MvcTagHelper_Order/Submit]]" method="HtmlEncode[[post]]">
```

Comment:
hmm, we over-encode generated URLs as well -- even using HTML helpers.  lack of a customer complaint probably means we can ignore my "todo" item for now.